src/components/CommentsList/CommentsList.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import './CommentsList.scss';
+
+import { CreateItem } from '../CreateItem';
+import { observer, inject } from 'mobx-react';
+import { IStores } from '../../interfaces';
+
+import { TweetItem } from '../TweetItem';
+
+export interface ICommentsListProps {
+  stores?: IStores,
+}
+
+@inject("stores")
+@observer
+export class CommentsList extends React.Component<ICommentsListProps, {}> {
+  constructor(props: ICommentsListProps) {
+    super(props);
+  }
+
+  componentWillMount() {
```

Comment:
Async actions should be inside `componentDidMount()`

https://medium.com/coffee-and-codes/componentdidmount-v-s-componnetwillmount-react-47f4f631276c